packages/components/src/index.ts

```diff
@@ -6,3 +6,4 @@ export * from './components/Markdown';
 export * from './components/Search';
 export * from './components/Menu';
 export * from './components/Drawer';
+export * from './components/List/index';
```

Comment:
I don't know if it's a good idea to export it on root.

Reply:
Remove the root export and leave only the component index, as it was being exported in packeage and tsup it continued to work, what do you think?